### DISCUSSION TITLE
Update from 0.15.1

### DISCUSSION THREAD

**Discussion by superpoli** [Stirling-Tools/Stirling-PDF#2375] [General]
hi,
my stirling 0.15.1 runs on an ubuntu server 22.04 under proxmox. How can I update it via console?
sudo apt-get update 
sudo apt-get upgrade
does not allow 7 packages to be updated.

**Comment by Frooodle** [comment]
You will have to download the new 
Jar file from releases and replace it

**Reply by superpoli** [reply]
Where can I find the jar and in which folder do I have to replace the file?

**Reply by Frooodle** [reply]
Wherever you placed it, if it's promox it's based on your own installation

**Reply by superpoli** [reply]
The whole thing is installed in a docker container. according to these instructions "https://kwellkorn.de/2024/09/21/stirling-pdf-im-proxmox-container-lxc/" Where can I find the jar file?

**Reply by Frooodle** [reply]
If it's docker then you just need to update the docker images following any docker updates instructions online

**Reply by superpoli** [reply]
ok thanks, it work now 0.35.1
